packages/radix-vue/src/ScrollArea/ScrollAreaScrollbarImpl.vue

```diff
@@ -128,5 +130,5 @@ useResizeObserver(rootContext?.content, handleSizeChange);
     @pointerup="handlePointerUp"
```

Comment:
we need to add `as` as injectedValue in `SCROLL_AREA_SCROLLBAR_INJECTION_KEY`.

then we can pass in the correct `:as="scrollbarContext?.as"` here

Reply:
[178e90ab59f8a4c659e2a600e0e31e09f06369f7](https://github.com/radix-vue/radix-vue/commit/178e90ab59f8a4c659e2a600e0e31e09f06369f7)